devices.json

```diff
@@ -528,6 +528,42 @@
         "type": "emulated-device",
         "device": {
             "show-by-default": false,
+            "title": "Microsoft Lumia 950",
+            "screen": {
+                "horizontal": {
+                    "width": 640,
+                    "height": 360 
+                },
+                "device-pixel-ratio": 4,
+                "vertical": {
+                    "width": 360,
+                    "height": 640
+                }
+            },
+            "capabilities": [
+                "touch",
+                "mobile"
+            ],
+            "user-agent": "Mozilla/5.0 (Windows Phone 10.0; Android 4.2.1; Microsoft; Lumia 950) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2486.0 Mobile Safari/537.36 Edge/13.10587",
```

Comment:
Yea there was a couple of other issues with the UA but I believe that the latest update is the "right one" (tm) and I've checked it with folk.